http-gateways/SUBDOMAIN_GATEWAY.md

```diff
@@ -0,0 +1,196 @@
+# Subdomain Gateway Specification
+
+![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
+
+**Authors**:
+
+- Marcin Rataj ([@lidel](https://github.com/lidel))
+
+----
+
+**Abstract**
+
+Subdomain Gateway is an extension of  [PATH_GATEWAY.md](./PATH_GATEWAY.md) that
+enables website hosting compatible with web browsers relative pathing and
+security model of the web. Below should be read as a delta on top of that spec.
```

Comment:
> compatible with web browsers relative pathing and security model of the web

This is the case for path gateway as well.

The main addition of subdomain gateway is to provide CID and IPNS names with their own environment following cross-origin isolation policy developed by web browsers

Possibly
```suggestion
Subdomain Gateway is an extension of  [PATH_GATEWAY.md](./PATH_GATEWAY.md) that
enables website hosting isolated per CID/name, while remaining compatible with web browsers
relative pathing and security model of the web
```